utils.go

```diff
@@ -528,11 +528,17 @@ func securityProxySetTLSCertificate(tlsCertificate, tlsPrivateKey, tlsSNI string
 	if err != nil {
 		return err
 	}
+
+	kongAdminToken, err := securityProxyReadFile(kongAdminTokenFile)
+	if err != nil {
+		return err
+	}
+
 	if tlsSNI != "" {
-		args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename, "--snis", tlsSNI}
+		args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename, "--admin_api_jwt", kongAdminToken, "--snis", tlsSNI}
 		err = securityProxyExecSecretsConfig(args)
 	} else {
-		args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename}
+		args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename, "--admin_api_jwt", kongAdminToken}
 		err = securityProxyExecSecretsConfig(args)
 	}
```

Comment:
```go
args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename, "--admin_api_jwt", kongAdminToken}
if tlsSNI != "" {
		args = append(args, []string{"--snis", tlsSNI}...)
}
err = securityProxyExecSecretsConfig(args)
```

Reply:
or better:
```go
args := []string{"proxy", "tls", "--incert", tlsCertFilename, "--inkey", tlsPrivateKeyFilename, "--admin_api_jwt", kongAdminToken}
if tlsSNI != "" {
		args = append(args, "--snis", tlsSNI)
}
err = securityProxyExecSecretsConfig(args)
```

Reply:
Thanks - I've updated the code and pushed the changes